src/rust/engine/process_execution/src/local.rs

```diff
@@ -36,12 +39,14 @@ impl CommandRunner {
     executor: task_executor::Executor,
     work_dir: PathBuf,
     cleanup_local_dirs: bool,
+    platform: Platform,
```

Comment:
This seems a strange thing to take as a parameter, rather than hard-coding in the constructor to `Platform::current_platform()`

Reply:
good idea.